x12-edi/2.5/modules/ROOT/pages/x12-edi-connector-hipaa.adoc

```diff
@@ -47,16 +47,16 @@ For example, if you've put your HIPAA `837` schema under `src/main/app/mypartner
 
 == About HIPAA Schema Definitions
 
-X12 Connector uses a YAML format called EDI Schema Language (ESL) to represent both standard X12 and HIPAA schemas.
+X12 EDI Connector uses a YAML format called EDI Schema Language (ESL) to represent both standard X12 and HIPAA schemas.
 The HIPAA standards are based on X12, but modify the corresponding X12 base standards in several aspects including:
 
 * Multiple specialized definitions of Hierarchical Level (HL) loops. This is where a basic loop structure defined in the X12 structure is expanded into several variations. These loops are often nested.
 
-* Modular loop definitions that can be reused at different points in the definition
+* Modular loop definitions that can be reused at different points in the definition.
 
-* Distinguishing between different uses of a repeated segment by code values in a particular element
+* Distinguishing between different uses of a repeated segment by code values in a particular element.
 
-* Changing usage requirements for segments and segment components
+* Changing usage requirements for segments and segment components.
```

Comment:
No periods, based on reformatting the first list item to our standard bullet-list style.